app/forms/remote_search_form.rb

```diff
@@ -0,0 +1,13 @@
+class RemoteSearchForm
+  include ActiveModel::Model
+  include ActiveModel::Validations::Callbacks
+
+  attr_accessor :online_checkbox, :by_phone_checkbox,
+    :pension_transfer,
+    :checkbox,
```

Comment:
`:checkbox` only a placeholder to support advisory filter fields; will remove it as a part of Scenario #4 from #5725